modules/gdscript/gdscript_analyzer.cpp

```diff
@@ -1925,29 +1922,41 @@ void GDScriptAnalyzer::resolve_return(GDScriptParser::ReturnNode *p_return) {
 		}
 		result = p_return->return_value->get_datatype();
 	} else {
-		// Return type is null by default.
```

Comment:
Any reason to remove this comment as well?

Reply:
Next lines obviously set things to be null, so the comment does not add anything. Does it?

Reply:
Put the comment back. Do not agree that there is any value in it, but ok.

Reply:
Even if that's the case, removing (non-incorrect) comments in code you're not changing just generates noise and isn't helpful.

Reply:
I consider myself updating whole `resolve_return` in this PR. The function is small, code before and after was changed.

Even if all people were in agreement that the comment is meh, it still would be awkward for me to open a separate PR specifically about removing a comment, useless change by itself.